UnityProject/Assets/Scripts/Tilemaps/Behaviours/Objects/RegisterTile.cs

```diff
@@ -305,6 +305,12 @@ public void OnDestroy()
 
 	public virtual void OnDespawnServer(DespawnInfo info)
 	{
+		if (objectLayer)
+		{
+			objectLayer.ServerObjects.Remove(LocalPositionServer, this);
+			objectLayer.ClientObjects.Remove(LocalPositionClient, this);
+		}
+
```

Comment:
Only called on server, wont affect clients

Reply:
will look into more, dam I really hate that need to save scenes cancel build thing

Reply:
isn't the problem server side?